### PR TITLE
add sample app to demo

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1162

### Change description ###

Add sample app to demo

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/demo/common/bsp/identity.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentityBinding
+metadata:
+  name: bsp-binding
+  namespace: bsp
+spec:
+  AzureIdentity: bsp
+  Selector: bsp
```
Comment (doncem) [inline k8s/demo/common/bsp/identity.yaml:20]: was this generated by script provided in bin folder?

Comment (mdayican) [reply k8s/demo/common/bsp/identity.yaml:20]: no, script  does not have demo, copied from portal.

Comment (doncem) [reply k8s/demo/common/bsp/identity.yaml:20]: oh is it not? sad

File: CODEOWNERS [hunk 2]
```diff
@@ -204,7 +204,7 @@ k8s/**/common/bsp/identity.yaml @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-payment-processor.yaml @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-processor.yaml @hmcts/bulk-scan
 k8s/**/common/bsp/bulk-scan-orchestrator.yaml @hmcts/bulk-scan
-k8s/aat/common/bsp/bulk-scan-sample-app.yaml @hmcts/bulk-scan
```
Comment (adusumillipraveen) [inline CODEOWNERS:207]: this is giving u access to deploy this to prod. so, pls revert.

Comment (doncem) [reply CODEOWNERS:207]: sample should not go to prod at all. but yea, perhaps worth just enlisting to be sure. unless there is some sort of _ignore_ list to override this setting

**Pull request merged**